### DISCUSSION TITLE
Interacting with Bokeh selections

### DISCUSSION THREAD

**Discussion by reesehopkins** [datapane/datapane#372] [General]
Hi,
I'm new to datapane and considering it for a project. I need to create an interactive chart where a user can select points on a scatter plot and have those selected points appear in a table. I like bokeh's [ColumnDataSource selected](https://docs.bokeh.org/en/latest/docs/reference/models/sources.html) concept. Is it possible to interact with this in datapane? Below is the closest relevant example I could find. It uses an on_change event. I only need to interact with the `source.selected.indices`

**Code**: https://github.com/bokeh/bokeh/blob/main/examples/server/app/selection_histogram.py
**Demo**: https://demo.bokeh.org/selection_histogram

Thanks!

**Comment by mands** [comment]
Hi @reesehopkins - sorry for the delay! Unfortunately Datapane doesn't provide such low-level access to the plot APIs after that have been initially created. You may have to work with Bokeh in Jupyter to work with `on_change` events.

We have an App API that is currently under development that allows for creating interactive data app experiments and may be useful - see https://docs.datapane.com for more info.
